src/docfx/build/Build.cs

```diff
@@ -129,6 +129,7 @@ private static void BuildFile(Context context, FilePath path)
 
                 context.ErrorLog.Write(errors);
                 Telemetry.TrackBuildFileTypeCount(file);
+                context.ContentValidator.ValidateManifest(file);
```

Comment:
directly return errors? so the telemetry can also track errors returned from validation lib